pkg/koordlet/util/system/version.go

```diff
@@ -40,14 +40,19 @@ func isAnolisOS() bool {
 }
 
 func isSupportBvtOrWmarRatio() bool {
-	bvtFilePath := filepath.Join(Conf.CgroupRootDir, CgroupCPUDir, CPUBVTWarpNsName)
+	var bvtFilePath, wmarkRatioPath string
+	bvtFilePath = filepath.Join(Conf.CgroupRootDir, CgroupCPUDir, CPUBVTWarpNsName)
```

Comment:
Do we still need to check the Anolis kernel for PSI on cgroups-v2? If we had to check the os version, we could instead use `GetCgroupResource()` to get the bvt and wmarkRatio resource on the proper cgroup version.

Reply:
PSI supports cgroups-v1 only on Anolis kernel, while supports cgroups-v1 & v2 on both Anolis kernel and community linux kernel.

Reply:
In the current implementation, the PSI collector will skip when the os version is not Anolis OS, no matter which cgroup version is.